app/pandas.ipynb

```diff
@@ -333,114 +284,123 @@
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
     "s1 = pd.Series([1, 2, 3, 4, 5])\n",
     "s2 = pd.Series([6, 7, 8, 9, 10])"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use + operator here"
+    "# use + operator here\n",
+    "s1 + s2"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use - operator here"
+    "# use - operator here\n",
+    "s1 - s2"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use * operator here"
+    "# use * operator here\n",
+    "s1 * s2"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use / operator here"
+    "# use / operator here\n",
+    "s1 / s2"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use ** operator here"
+    "# use ** operator here\n",
+    "s1 ** s2"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use // operator here"
+    "# use // operator here\n",
+    "s1 // s2"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# use % operator here"
+    "# use % operator here\n",
+    "s1 % s2"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# filter s1 here"
+    "# filter s1 here\n",
+    "s1[::2]"
    ],
    "metadata": {
     "collapsed": false
-   }
+   },
+   "outputs": [],
+   "execution_count": null
   },
   {
    "cell_type": "code",
-   "execution_count": null,
-   "outputs": [],
    "source": [
-    "# filter s2 here"
+    "# filter s2 here\n",
+    "s2[1::2]"
```

Comment:
The filtering of odd numbers from `s2` is incorrect. You should filter using a condition like `s2[s2 % 2 != 0]` to get odd numbers.